x-pack/plugins/monitoring/public/components/no_data/__snapshots__/checker_errors.test.js.snap

```diff
@@ -26,7 +26,7 @@ Array [
         data-type="row"
       >
         <dt
-          class="euiDescriptionList__title emotion-euiDescriptionList__title-row-m-normal"
+          class="euiDescriptionList__title emotion-euiDescriptionList__title-row-normal-s"
```

Comment:
@weltenwort Correct, the snapshots updated because the default size was changed to `s`. This was a conscious decision by EUI to reduce the margin between rows on `EuiDescriptionList`. If there are layouts you want to retain the larger margin, they can be overrode by passing `rowGutterSize="m"` into the component.